docs/docs/guides/developer_guides/getting_started.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Getting Started
+sidebar_position: 0
+tags: [sandbox]
+---
+
+This guide will teach you how to install the Aztec sandbox, deploy your first smart contract, and interact with it using the CLI.
+
+The Sandbox is an Aztec network running fully on your machine, and interacting with a development Ethereum node. You can develop and deploy on it just like on a testnet or mainnet. 
+
+## Prerequisites
+
+You need two global dependencies iyour machine:
+
+- Node.js >= v18 (recommend installing with [nvm](https://github.com/nvm-sh/nvm))
+- Docker (visit [this page of the Docker docs](https://docs.docker.com/get-docker/) on how to install it)
+
+## Install and run the sandbox
+
+**Start Docker**
+
+Docker needs to be running in order to install the sandbox. Find instructions on the [Docker website](https://docs.docker.com/get-started/).
+
+**Install the sandbox**
+
+Run:
+
+```bash
+bash -i <(curl -s https://install.aztec.network)
+```
+
+This will install the following tools:
+
+- **aztec** - launches various infrastructure subsystems (full sandbox, sequencer, prover, pxe, etc) and provides utility commands to interact with the network
+- **aztec-nargo** - aztec's build of nargo, the noir compiler toolchain.
+- **aztec-up** - a tool to upgrade the aztec toolchain to the latest, or specific versions.
+
+**Start the sandbox**
+
+Once these have been installed, to start the sandbox, run:
+
+```bash
+aztec start --sandbox
+```
+
+**Congratulations, you have just installed and run the Aztec Sandbox!**
+
+```bash
+     /\        | |
+    /  \    ___| |_ ___  ___
+   / /\ \  |_  / __/ _ \/ __|
+  / ____ \  / /| ||  __/ (__
+ /_/___ \_\/___|\__\___|\___|
+
+```
+
+In the terminal, you will see some logs:
+
+1. Sandbox version
+2. Contract addresses of rollup contracts
+3. PXE (private execution environment) setup logs
+4. Initial accounts that are shipped with the sandbox and can be used in tests
+
+You'll know the sandbox is ready to go when you see something like this:
+
+```bash
+[INFO] Aztec Server listening on port 8080
+```
+
+## Creating an account in the sandbox
+
+Now you have the sandbox running, let's create an account. Open a new terminal window and run:
+
+```bash
+aztec-wallet create-account
```

Comment:
You can pass the `-a` flag to make it an alias like:

```bash
aztec-wallet create-account -a my_account
```

And then just reference this account like `accounts:my_account`. This is the correct way of doing as it already takes care of all the required fields for any subsequent command, and avoids having to `export` stuff.

I think the `CLI` wallet tutorial could help with this. @Thunkar is the mastermind behind aliases